app/controllers/health.controller.js

```
@@ -1,164 +1,80 @@
+											'Go to the resources tab and select Compose MongoDB. In the Compose interface, go to the admin tab and trigger a primary step-down ' +
+											'by clicking on the Trigger Step-down button in the Change primary section. ' +
+											'Wait a few minutes to see if the step down has fixed the issue. If not, go back to resources tab on the Heroku console, ' +
+											'set all the dynos to 0, and reset them after a few seconds to their original value.',
+					lastUpdated: new Date().toISOString(),
+					ok: dbStatusResponse.up
+				},
+				{
+					name: 'Queue is Up',
+					id: 'email-simple-send-queue-check',
+					severity: 1,
+					businessImpact: 'Will not be able to send emails via simple send service',
+					technicalSummary: 'Pings an endpoint which publishes and consumes a test message to ensure correct connectivity and functionality',
+					panicGuide: 'First, verify that there is not a global issue with CloudAMQP by visiting http://status.cloudamqp.com. ' +
+											'If there is a global issue with CloudAMQP, then no further action can be taken to fix the issue. ' +
+											'If there is no CloudAMQP issue, visit the Heroku dashboard for ft-email-simple at https://dashboard.heroku.com/apps/ft-email-simple/resources. ' +
+											'Go to the resources tab and select CloudAQMP. In the CloudAMQP interface, go to the RabbitMQ nodes tab ' +
+											'and reboot the server instances by clicking the Reboot server instance button on all of the listed nodes. ' +
+											'Wait a few minutes to see if the reboot has fixed the issue. If not, go back to resources tab on the Heroku console, ' +
+											'set all the dynos to 0, and reset them after a few seconds to their original value.',
+					lastUpdated: new Date().toISOString(),
+					ok: queueStatusResponse.up
+				}
+			]
+		});
+	} catch (err) {
+		// eslint-disable-next-line no-console
+		console.log(err);
```

Should we log this somewhere?

---

Ooh yep, if we use [our logger util](https://github.com/Financial-Times/email-send-api/blob/master/logger.js) it’ll always go to Splunk if you call `logger.error`. Best wrap it in another try/catch though (I know, bleurgh)—we want health checks to be as bullet proof as possible.

---

Sure 👍 

So wrap `logger.error` in a `try/catch`.  What should we do in the `catch` case? Can we fail silently?

---

Maybe still a console.log as a last resort

---

Fixed: https://github.com/Financial-Times/email-send-api/pull/258/commits/9e5e1437e4e76fb59b56723035e8f935e347447e 👍